Asterix/Optical_System_functions.py

```diff
@@ -424,23 +444,25 @@ def generate_phase_aberr(self, SIMUconfig, Model_local_dir=None):
         ## Phase map and amplitude map for the static aberrations
         if set_phase_abb == True:
             if phase_abb_filename == '':
-                phase_abb_filename = "phase_{:d}rms_spd{:d}_rhoc{:.1f}_rad{:d}".format(
+                phase_abb_filename = "opd_{:d}rms_spd{:d}_rhoc{:.1f}_rad{:d}".format(
                     int(phase_rms * 1e9), int(phase_slope), phase_rhoc,
                     self.prad)
 
             if set_random_phase is False and os.path.isfile(
                     Model_local_dir + phase_abb_filename + ".fits") == True:
-                return_phase = fits.getdata(Model_local_dir +
-                                            phase_abb_filename + ".fits")
+                random_opd = fits.getdata(Model_local_dir +
+                                          phase_abb_filename + ".fits")
 
             else:
-                return_phase = phase_ampl.random_phase_map(
-                    self.prad, self.dim_overpad_pupil, phase_rms, phase_rhoc,
-                    phase_slope)
+                random_opd = phase_ampl.random_opd_map(self.prad,
```

Comment:
je ne suis vraiment pas convaincu du tout.

Si tu veux tout passer en OPD, ok mais alors il faut changer tous les noms de fonctions, d'arguments et de paramètres.
Car là, c'est vraiment difficile à lire random_opd est calculé avec random_opd_map de phase_ampl et les arguments sont phase_rms et phase_rhoc et phase_slope...
Je sais que ce n'était pas parfait avant mais cela me semblait moins mélangé (dans mon souvenir).

Reply:
- I renamed random_opd_map in random_phase_map 
- I changed phase_rms to opd_rms in the parameter file and transformed the opd_rms before using random_phase_map